### PR TITLE
Proxy support

### PR DESCRIPTION
**Pull request by mahirchavda**

### CODE REVIEW

File: TA-microsoft-graph-security-score/app.manifest [hunk 1]
```diff
@@ -1,62 +1,59 @@
...
-      "uri": null
-    },
-    "privacyPolicy": {
-      "name": null,
-      "text": null,
-      "uri": null
+    "inputGroups": {},
+    "platformRequirements": {
+        "splunk": {
+            "Enterprise": "*"
```
Comment (VatsalJagani) [inline TA-microsoft-graph-security-score/app.manifest:45]: Shall we include Cloud here as well?

**Review by VatsalJagani — APPROVED** [review]

**Comment by VatsalJagani** [comment]
There is no change in calling APIs right?

File: TA-microsoft-graph-security-score/app.manifest (hunk 1 above)
Comment (mahirchavda) [reply TA-microsoft-graph-security-score/app.manifest:45]: It is fine. Maxmind app has same config.

**Comment by mahirchavda** [comment]
> There is no change in calling APIs right?

Right no change in the api call. Since we are using ucc library code for api call. It handles proxy logic by default

**Review by hardikhdholariya — APPROVED** [review]

**Pull request merged**